homeassistant/components/comfoconnect/fan.py

```diff
@@ -44,12 +45,18 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class ComfoConnectFan(FanEntity):
     """Representation of the ComfoConnect fan platform."""
 
+    _attr_icon = "mdi:air-conditioner"
+    _attr_should_poll = False
+    _attr_supported_features = SUPPORT_SET_SPEED
```

Comment:
Fans do not support this feature yet, as per developer documentation:

<https://developers.home-assistant.io/docs/core/entity/fan#properties>

This platform should not be using these.